app/assets/javascripts/3dbio_viewer/src/domain/entities/__tests__/PdbInfo.spec.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import { describe, expect, it } from "@jest/globals";
+import { getCompositionRoot } from "../../../compositionRoot";
+import { defaultPdbId } from "../../../webapp/pages/app/AppRouter";
+
+describe("PdbInfo", () => {
+    it("with PDB 6zow to be truthy", async () => {
+        axios.defaults.adapter = require("axios/lib/adapters/http");
```

Comment:
On next iterations, let's move this to some init / `before` block, so it's not repeated in all tests.